src/main.rs

```diff
@@ -73,20 +77,47 @@ fn load_custom_certificate<P: AsRef<Path>>(path: P) -> Result<Certificate, Error
     Ok(cert)
 }
 
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "authd", about = "An authentication daemon.")]
+struct Opt {
+
+    #[structopt(long, default_value = ".ssl/ca-bundle.crt", parse(from_os_str))]
+    /// Path to a custom certificate for securing HTTP connections.
+    cert: PathBuf,
```

Comment:
I wonder if this should be optional. What do you think @gerred?

Reply:
If this were optional, would you generate the cert?

Reply:
Or would you just go over HTTP? idk, feels like forcing HTTPS is ok in 2019

Reply:
If no cert is given we would not add any to the client and it would use the root certs etc on the system.

Reply:
got it :+1: definitely make it optional